test/employees-service.test.ts

```diff
@@ -0,0 +1,105 @@
+import {
+    TestBed, fakeAsync, tick
+} from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { MockModule } from './mock.module';
+import { EmployeesService, createEmployee, employeeTitles } from '../src/employees.service';
+import { HttpClientModule } from '@angular/common/http';
+
+const fields = ['managerId', 'id', 'name', 'firstName', 'lastName', 'title', 'active', 'hireDate', 'phone'];
+const testHier = (data) => {
+        expect(data[0].reports[0].reports[1].reports[0]).toEqual(akeemEmployee);
+};
+const akeemEmployee = {"active": true, "firstName": "Akeem", "hireDate": new Date("2017-01-15"), "id": 60, "lastName": "Carr", "name": "Akeem Carr", "phone": 7381362814, "title": "Junior Software Developer"};
+
+describe('employees', () => {
+    @Component({
+        template: ''
+    })
+    class TestComponent {
+        constructor(public service: EmployeesService) {}
+    }
+    let service;
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [TestComponent],
+            imports: [MockModule,  HttpClientModule],
+            providers: [EmployeesService]
+        });
+        const template: string = '<kendo-component></kendo-component>';
```

Comment:
And the template?

Reply:
To harness the component, do not want to just test the service on functional level , it is also important that can be integrated with regular component. Do you want me to dig some examples from the current component's code ?